Revit_oM/Commands/DirectPush.cs

```diff
@@ -0,0 +1,49 @@
+/*
+ * This file is part of the Buildings and Habitats object Model (BHoM)
+ * Copyright (c) 2015 - 2025, the respective contributors. All rights reserved.
+ *
+ * Each contributor holds copyright over their respective contributions.
+ * The project versioning (Git) records all such contribution source information.
+ *                                           
+ *                                                                              
+ * The BHoM is free software: you can redistribute it and/or modify         
+ * it under the terms of the GNU Lesser General Public License as published by  
+ * the Free Software Foundation, either version 3.0 of the License, or          
+ * (at your option) any later version.                                          
+ *                                                                              
+ * The BHoM is distributed in the hope that it will be useful,              
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of               
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the                 
+ * GNU Lesser General Public License for more details.                          
+ *                                                                            
+ * You should have received a copy of the GNU Lesser General Public License     
+ * along with this code. If not, see <https://www.gnu.org/licenses/lgpl-3.0.html>.      
+ */
+
+using BH.oM.Base;
+using System;
+using System.Collections.Generic;
+using System.ComponentModel;
+
+namespace BH.oM.Adapter.Commands
+{
+    [Description("Push action allows quick transfer of selected objects on current UI to External Application.")]
+    public class DirectPush : IExecuteCommand
+    {
+        /***************************************************/
+        /****             Public Properties             ****/
+        /***************************************************/
+
+        [Description("Source objects to be pushed")]
+        public virtual List<IObject> Identifiers { get; set; }
```

Comment:
Here I would rename the property to something more explicit, `Objects`, `Elements`, `ObjectsToPush`? Also description is a bit vague 😉